web-share/share-null-manual.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>WebShare Test: Share null and undefined values</title>
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+    <script src="resources/manual-helper.js"></script>
+  </head>
+  <body>
+    <script>
+        setup({explicit_timeout: true});
+
+        // Expect null to be converted into the string 'null'. (This isn't good
+        // practice, but it's how Web IDL specifies conversion.) On the other
```

Comment:
> That was my initial reaction too but this isn't really a matter of opinion for an individual standard.
> (In summary: yes, it's weird and unintuitive, but very much normal.)

Yup. Agreed. :)

We used to have [\[TreatNullAs=EmptyString\]](https://heycam.github.io/webidl/#TreatNullAs) for that, but as you said, that ship has sailed.

While on that topic, have you considered making `""` the default value for some of these (and thus avoiding having to deal with dictionary members that are not present)?